src/routes/graphql.js

```diff
@@ -137,6 +138,7 @@ function createApolloServerOptions() {
         rootValue: null,
         schema,
         engine: engineEnabled ? engineConfig : false,
+        plugins: apolloPlugins ? [apolloPlugins] : [],
```

Comment:
Do we want to wrap this in an array? Might it make more sense for `apolloPlugins` to already be an array (it's plural, after all)?

```suggestion
        plugins: apolloPlugins ?? [],
```

Reply:
I will do this, since this is not used in a frontend app directly, right? Saying this, because I checked out of curiosity and it is not supported on Internet Explorer https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#browser_compatibility) @simpleigh-globality

Reply:
actually, i will look if it's appropriate to do this, because this would mean the apollo plugins would be wrapped

Reply:
Sorry: I've suggested two separate changes in the same comment here which wasn't particularly helpful:

1. Dropping the array wrapping so it's possible to pass in multiple `apolloPlugins` from outside:
    ```suggestion
            plugins: apolloPlugins ? apolloPlugins : [],
    ```
    If you don't do this then I think you should probably change the name to `apolloPlugin` since you only support one of them?
2. _If_ you make the change above, you can then use the `??` rather than the ternary (I hate ternaries 😄).

No more work on this tonight please 😉

Reply:
at first I thought that it had to be implemented like I did, but looking at documentation it's really best if i provide an array while setting defaults. https://www.apollographql.com/docs/apollo-server/integrations/plugins/#installing-a-plugin

Reply:
I managed to do this change, the tests pass, but yarn lint does not locally 
```
~/d/nodule-graphql GLOB-51227/a…tup-from-GWs !2 ❯ yarn lint                               11:52:28
yarn run v1.22.4
$ eslint src --cache

/Users/isabelcosta/dev/nodule-graphql/src/routes/graphql.js
  146:33  error  Parsing error: Unexpected token ?

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
i replaced for `||` which seems to work.
Also, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#assigning_a_default_value_to_a_variable

Reply:
Gah, stupid `eslint`.